test/telemetry.test.ts

```diff
@@ -80,12 +80,12 @@ describe('Telemetry Test', () => {
 
     it('should send telemetry if log error', () => {
       telemetryChannel.append('[Error Some');
-      expect(telemetry.send).calledOnceWith({ name: 'server_error', properties: { error_message: '[Error Some' } });
+      expect(telemetry.send).calledOnceWith({ name: 'yaml.server.error', properties: { error: '[Error Some' } });
     });
 
     it('should send telemetry if log error2', () => {
       telemetryChannel.appendLine('[Error Some');
-      expect(telemetry.send).calledOnceWith({ name: 'server_error', properties: { error_message: '[Error Some' } });
+      expect(telemetry.send).calledOnceWith({ name: 'yaml.server.error', properties: { error: '[Error Some' } });
```

Comment:
those are identical tests. what are you checking?

also my ocd kicks in when I see that missing closing bracket in `[Error Some`